src/Command/Command.php

```diff
@@ -247,7 +246,7 @@ public function bindValues(array $values): self
                 $this->params[$value->getName()] = $value;
             } elseif (is_array($value)) { // TODO: Drop in Yii 2.1
                 $this->params[$name] = new Param($name, ...$value);
-            } elseif ($value instanceof PdoValue) {
+            } elseif ($value instanceof PdoValue && is_int($value->getType())) {
```

Comment:
This incorrect.
https://www.php.net/manual/en/pdo.constants.php
// public const PARAM_NULL = 0;